openapi/mx_platform_api_beta.yml

```diff
@@ -2439,14 +2439,14 @@ paths:
               schema:
                 "$ref": "#/components/schemas/AccountOwnersResponseBody"
           description: OK
-      summary: List account owners
+      summary: List account owners by member
       tags:
       - mx_platform
   "/users/{user_guid}/members/{member_guid}/accounts/{account_guid}":
     put:
       description: This endpoint allows you to update certain attributes of an `account`
         resource.
-      operationId: updateAccount
+      operationId: updateAccountByMember
```

Comment:
This also seems confusing to me. We aren't just updating an account by a `member` but by a `user` as well.

Reply:
Addressed in the first comment above.